spec/tests/runner_spec.rb

```diff
@@ -43,8 +43,11 @@
     @dir = "#{Dir.pwd}/spec/outputs/#{building_type}"
```

Comment:
Should we rename the file to something else besides `runner_spec`? What is the exact intention?

Reply:
I'm not sure, I didn't make the test. It looks like it only exists to test whether the models can still run after the sensors are added by the translation process

Reply:
Yeah, ok, I remember that now. In the lines below, when you create a new translator from a model and mappings manager, does that save the model back to disk? Each of the `it 'MediumOffice can run an OSW'` portions expects to read the model from disk. Regardless of if it's reading the model from disk or just getting the model in memory, can you make sure that where the current `run_osw` parts are it is actually running the model that has the most recent updates?

Like you said, the intention of those tests is to make sure we don't break anything on the model side of things when we do the translation that prevents a simulation from being performed, so I just want to make sure that is actually happening and not a placebo.

Reply:
I just looked into this and it seems like it's not actually doing anything. As of right now the changes to the OS model are not saved after translation. I'm not quite sure how we actually want to handle this.

Reply:
I think we should probably just open an issue for this as it is not really within the scope of this PR

Reply:
yes, cool, please do